CHANGES.rst

```diff
@@ -8,11 +8,26 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to true on default.
```

Comment:
I think that `fetch` should be `False` by default. Otherwise, for some workflows, PRAW users will burn through their rate-limit twice as fast without an understanding of why.

Reply:
I see. Maybe we can default to true on a major release, because this really should be True by default. The average user doesn’t read through the notes and warnings, and this can and will prevent accidentally deleting flairs.

Reply:
I like the idea of defaulting to `True` on a major release.